components/View/FormattedTime.js

```diff
@@ -3,9 +3,8 @@ import React from 'react';
 
 import FormattedDate from './FormattedDate';
 
-// NOTE: capturing 'date', as it is deprecated in FormattedDate
 // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-const FormattedTime = ({ value, date, ...otherProps }) => {
+const FormattedTime = ({ value, ...otherProps }) => {
```

Comment:
Did you check the backward compatibility for this change?

Reply:
No